utilities/xmlrpcpp/src/XmlRpcDispatch.cpp

```diff
@@ -100,9 +105,9 @@ XmlRpcDispatch::work(double timeout)
   while (_sources.size() > 0) {
 
     // Construct the sets of descriptors we are interested in
-    const unsigned source_cnt = _sources.size();
-    pollfd fds[source_cnt];
-    XmlRpcSource * sources[source_cnt];
+    unsigned source_cnt = _sources.size();
```

Comment:
Why can't this variable be `const` anymore?

Reply:
on MSVC compiler will complain that _sources.size() is not an constant, so I need to remove the const.  I'll investigate to see if we can add it back